src/components/SchedulePage/Schedule.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import TimeSlot from '../SchedulePage/Timeslot';
+
+// Must be ordered
+const testEvents = [
+	{
+		name: 'ACM-W X ACM Hack | Hackathon 101 + Team Formation',
+		location: 'Sycamore Room',
+		// eslint-disable-next-line no-multi-str
+		description: 'First hackathon? We\'ll show you all the tips and tricks you\'ll ' +
+		'need to know to be successful and most importantly, have fun!',
+		startTime: '9:30 AM'
+	},
+	{
+		name: 'Git and Github',
+		location: 'De Neve Auditorium',
+		// eslint-disable-next-line no-multi-str
+		description: 'With multiple developers each working on ' +
+		'a different feature of a project, bringing things together ' +
+		'can get messy pretty easily. Come learn how to use Git to ' +
+		'manage code across multiple collaborators and versions!',
```

Comment:
I feel like we should avoid disabling eslint's no-multi-str, but I'm not sure what we would replace it with. 

I think we could instead use a React Fragment, i.e.
``` javascript
description: <>With multiple developers each working on
		a different feature of a project, bringing things together
		can get messy pretty easily. Come learn how to use Git to
		manage code across multiple collaborators and versions!</>
```
but I'd prefer for @TimothyGu to weigh in.

Reply:
Hmmm, I'll experiment with this. Seems clever. I didn't think to try this. I'll be back with results.

Reply:
This looks like it works. We can always undo it if they're issues.

Reply:
I agree with @jamieliu386 that we should avoid overriding ESLint whenever possible.

Generally as a rule, I'd rather not use JSX with text. There are some known pitfalls with doing so, like https://stackoverflow.com/questions/40264084/best-practice-when-adding-whitespace-in-jsx.

How about using a template literal instead?
```js
		description: `With multiple developers each working on
		a different feature of a project, bringing things together
		can get messy pretty easily. Come learn how to use Git to
		manage code across multiple collaborators and versions!`,
```

Reply:
(Of course, my recommendation only applies if you don't have to create additional elements.)

Reply:
@TimothyGu We can't have line breaks then, but that seems like a sacrifice that I'd be okay with making.

Reply:
Feel free to override this if you need to create additional elements. Also, see my other comment: we probably should not be using `<br>` anyway.